### ISSUE TITLE
HCStumping turns logs you place on Grass into stumps?

### ISSUE THREAD

**Issue by Dylan4ever** [BetterWithMods/BetterWithMods#881]
Using 2.1.12, when I place a log on Grass or Dirt, it turns into a stump. Is this the intended behaviour? If you misplace a log for decorative purposes, for example, this could cause great annoyance.

**Comment by primetoxinz** [comment]
Yeah, this is unfortunately consequence of how we make stumps work for any log in the game. There is no way to detect if the block was world-generated or placed by a player.
